docs/user-guide/prereqs.md

```diff
@@ -21,11 +21,30 @@ certainly want the docs that go with that version.</h1>
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Kubernetes User Guide: Managing Applications: Prerequisites
+To deploy and manage applications on Kubernetes, you’ll use the Kubernetes command-line tool, [kubectl](kubectl/kubectl.md). It lets you inspect your cluster resources, create, delete, and update components, and much more. You will use it to look at your new cluster and bring up example apps. 
 
-To deploy and manage applications on Kubernetes, you’ll use the Kubernetes command-line tool, [kubectl](kubectl/kubectl.md). It can be found in the release tar bundle, or can be built from source from github. Ensure that it is executable and in your path.
+##Install kubectl
+You can find it in the [release](https://github.com/GoogleCloudPlatform/kubernetes/releases) tar bundle, under platforms/<os>/<arch>;
+or if you build from source, kubectl should be either under _output/local/bin/<os>/<arch> or _output/dockerized/bin/<os>/<arch>.
 
+Next, make sure the kubectl tool is in your path, assuming you download a release:
+```
+# OS X
+export PATH=<path/to/kubernetes-directory>/platforms/darwin/amd64:$PATH
+
+# Linux
+export PATH=<path/to/kubernetes-directory>/platforms/linux/amd64:$PATH
+```
+
+An alternate method, useful if you're building from source and do not want to configure the PATH, is using ./cluster/kubectl.sh instead of kubectl. That script will auto-detect the location of kubectl and proxy commands to it (eg., ./cluster/kubectl.sh cluster-info).
```

Comment:
Would prefer we not direct users to this method. We want to stop using cluster/kubectl.sh as a command proxy and instead have it find the binary location and `export PATH=...`